docs/neurobagel_environment_variables.tsv

```diff
@@ -20,5 +20,6 @@ Environment variable	Set manually in .env?	Description	Default value if not set
 `NB_QUERY_TAG`	No	Docker image tag for the query tool	`latest`	Docker
 `NB_QUERY_PORT_HOST`	No	Port number used by the `query_tool` on the host machine	`3000`	Docker
 `NB_FEDERATE_REMOTE_PUBLIC_NODES`	Yes 	If "True", include public nodes in federation	`true`	Docker, Python
+`NB_QUERY_URL_PATH`	Yes	The URL path for the query tool, determines the specific URL at which the app should be rendered for users to access it	`/`	Docker
```

Comment:
🍒 :
```suggestion
`NB_QUERY_URL_PATH`	No	The URL path for the query tool, determines the specific URL at which the app should be rendered for users to access it	`/`	Docker
```

The language of the column is definitely a bit confusing (another item to address in https://github.com/neurobagel/recipes/issues/30?), but I think we should reserve `Set manually in env?` to be `Yes` for variables that we either don't have a default for, like passwords, or variables we want to users to look at & change from the default, for security or usability purposes. For the sake of an easy setup, and the fact that most users probably won't use something other than the default for `NB_QUERY_URL_PATH` immediately, I think we can keep this a `No`. Let me know if you disagree!